src/components/entity-highlighter/EntityHighlighter.js

```diff
@@ -0,0 +1,53 @@
+import React, { useRef, useState } from 'react';
+import { TextArea } from './TextArea';
+import { AddEntityInput } from './AddEntityInput';
+import { FoundEntities } from './FoundEntities';
+
+export const EntityHighlighter = () => {
+  const [{selectionStart, selectionEnd}, setSelection] = useState({ selectionStart: 0, selectionEnd: 0});
+  const [text, setText] = useState('Venture first mover advantage learning curve market ecosystem funding stealth disruptive social proof scrum project growth hacking niche market user experience graphical user interface.');
+  const [entities, setEntities] = useState([
+    { start: 160, end: 184, label: 'very important'},
+    { start: 144, end: 159, label: 'very important'},
+    { start: 62, end: 69, label: 'important' },
+    { start: 116, end: 130, label: 'nonsense' },
+    { start: 8, end: 29, label: 'nonsense' },
+  ]);
```

Comment:
Three pieces of state that need to "live" in this level as they are either passed down to multiple children components, used in methods passed to children components or used to calculate values passed to children.